lib/mutant/cache.rb

```diff
@@ -21,7 +24,7 @@ def initialize
     # @api private
     def parse(path)
       @cache.fetch(path) do
-        @cache[path] = Parser::CurrentRuby.parse(File.read(path))
+        @cache[path] = Parser::CurrentRuby.parse(file.read(path))
```

Comment:
So this `file` object is actually a `::File` class? Why not have your `path` object be a `Pathname`? That's better than passing around `String` objects to represent a file path.

IMHO I would find wherever these `String` objects are created and turn them into `Pathname` objects. A `String` primitive is rarely the correct object; except maybe where you are dealing with opaque text that is getting logged, certainly not the case here.